dataactcore/models/base.py

```diff
@@ -0,0 +1,4 @@
+from sqlalchemy.ext.declarative import declarative_base
+from dataactcore.utils.timeStampMixin import TimeStampBase
+
+Base = declarative_base(cls=TimeStampBase)
```

Comment:
Yep, it's specifically for the validation database--I needed to extract out the Base stuff now that we're importing more than one file into validationModels (stagingModels now, in addition to domainModels). 

Good call on being more specific in the name--just pushed a change for that.